compiler/rustc_session/src/session.rs

```diff
@@ -1515,12 +1491,6 @@ pub fn build_session(
         io,
         incr_comp_session: OneThread::new(RefCell::new(IncrCompSession::NotInitialized)),
         prof,
-        perf_stats: PerfStats {
```

Comment:
definition `pub struct PerfStats` become dead code too?

Reply:
Good catch. I have updated to remove it.